permafrost/models.py

```diff
@@ -24,6 +24,20 @@
 # CHOICES
 ###############
 
+try:
+    PERMAFROST_DEFAULT_ROLES = getattr(settings, 'PERMAFROST_DEFAULT_ROLES')
+except AttributeError:
+    PERMAFROST_DEFAULT_ROLES = [
```

Comment:
wouldn't you add this in the settings.py file?

Reply:
yea it needs to be added to the tc settings file, While testing I needed this default to make sure it actually works